syfoalerts.yaml

```diff
@@ -11,21 +11,21 @@ spec:
       #prependText: '<!here> | '
   alerts:
     - alert: syfo-app-nede
-      expr: up{app=~"syfoarbeidsgivertilgang|syfonarmesteleder|syfoservicestrangler|syfosoknad|syfosyketilfelle|syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy",job="kubernetes-pods"} == 0
+      expr: up{app=~"syfoarbeidsgivertilgang|syfonarmesteleder|syfoservicestrangler|syfosoknad|syfosyketilfelle|syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy|syfosmapprec",job="kubernetes-pods"} == 0
```

Comment:
Er det noen som har synkrone avhengigheter til syfosmapprec? Denne alerten er kun ment for de appene som andre har synkrone avhengigheter til, så den vil fyre ganske raskt :)

Reply:
Vi bør iallefall ikke ha samme app i både denne alerten og den for asynkron :)

Reply:
Nope, ingen synkrone avhengigheter i fix